sklearn/tree/_criterion.pyx

```diff
@@ -829,7 +834,8 @@ cdef class RegressionCriterion(Criterion):
 
                 for k in range(self.n_outputs):
                     y_ik = y[i * self.y_stride + k]
-                    sum_left[k] += w * y_ik
+                    w_y_ik = w * y_ik
+                    sum_left[k] += w_y_ik
```

Comment:
Ah oops, that was when I was trying things out. Will revert.

On Thu, Jun 23, 2016, 3:06 PM Gilles Louppe <notifications@github.com>
wrote:

> In sklearn/tree/_criterion.pyx
> <https://github.com/scikit-learn/scikit-learn/pull/6667#discussion_r68184637>
> :
>
> > @@ -829,7 +834,8 @@ cdef class RegressionCriterion(Criterion):
> >
> >                  for k in range(self.n_outputs):
> >                      y_ik = y[i * self.y_stride + k]
> > -                    sum_left[k] += w * y_ik
> > +                    w_y_ik = w * y_ik
> > +                    sum_left[k] += w_y_ik
>
> Is this micro-change really necessary?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6667/files/20d6107c7ebfba1b96d483cfacb97a9ac304b3cb#r68184637>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG72X1ip3jUnpAL3fJHMHQF90yaTsCLMks5qOjBagaJpZM4IIwnV>
> .
>